setup.py

```diff
@@ -24,5 +24,8 @@
         ],
     },
     url='https://github.com/broamski/aws-mfa',
-    install_requires=['boto3']
+    install_requires=[
```

Comment:
aren't `'dbus-python', 'secretstorage'` required as well to support D-Bus?

Reply:
Resolved in f558bc5bb5c367e6071baada46a5567db24c6d07